src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -826,4 +831,55 @@ public static Method getIbdoMethod(final String methodName, final Class<?>... pa
             throws NoSuchMethodException {
         return IBaseDataObject.class.getMethod(methodName, parameterTypes);
     }
+
+    public static boolean requiresEncoding(final byte[] utf8Bytes) {
```

Comment:
Any reason not to update the ByteUtil or add these methods to a util class?? Also, since these are public methods, I'd like to see some javadocs.